modules/services/megasync.nix

```
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.megasync;
+
+in {
+  meta.maintainers = [ maintainers.GaetanLepage ];
+
+  options = {
+    services.megasync = {
+      enable = mkEnableOption "Megasync client";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.megasync;
+        defaultText = literalExpression "pkgs.megasync";
+        description = "The package to use for the megasync binary.";
+      };
```

```suggestion
      package = mkPackageOption pkgs "megasync" { };
```

You could simplify the `package` option using `mkPackageOption`.

---

Done :)